config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/tasks/zip/builder/ListenPortEntityBuilder.java

```diff
@@ -101,20 +101,18 @@
     public List<Entity> build(Bundle bundle, BundleType bundleType, Document document) {
         switch (bundleType) {
             case DEPLOYMENT:
-                final Stream<Entity> userPorts = bundle.getListenPorts().entrySet().stream()
-                        .map(listenPortEntry -> buildListenPortEntity(bundle, listenPortEntry.getKey(), listenPortEntry.getValue(), document));
+                return bundle.getListenPorts().entrySet().stream()
```

Comment:
There's some code duplication here, maybe we can make the userports stream above this switch statement?